.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+    ]
```

Comment:
add plugins which will be required 
eg: "@babel/plugin-proposal-class-properties", for addictively class properties